controller/backup_controller.go

```diff
@@ -448,39 +447,41 @@ func (bc *BackupController) backupCreation(log logrus.FieldLogger, engineClient
 		biChecksum = bi.Status.Checksum
 	}
 
-	backup.Status.State = types.BackupStatePending
-	logEvent(nil)
+	backup.Status.State = types.BackupStateInProgress
```

Comment:
After removing state `pending`, how can this controller avoid duplicate backup calls?

Reply:
If the state is not empty, we won't call engineapi.snapshotCreate because we think the backup creation called once already.